src/lib.rs

```diff
@@ -13,10 +13,12 @@ mod constants;
 mod ixgbe;
 pub mod memory;
 mod pci;
+pub mod interrupts;
```

Comment:
Should this module be public? And do the constants, structs and methods in the module have to be public? If not, please make them private or `pub(crate)`